api/src/test/java/uk/gov/dhsc/htbhf/claimant/ScheduledServiceIntegrationTest.java

```diff
@@ -109,6 +109,13 @@ void assertThatChildTurnsFourEmailWasSent(PaymentCycle currentCycle) throws Noti
         assertChildTurnsFourEmailPersonalisationMap(currentCycle, argumentCaptor.getValue());
     }
 
+    void assertThatPaymentStoppingEmailWasSent(PaymentCycle currentCycle) throws NotificationClientException {
+        ArgumentCaptor<Map> argumentCaptor = ArgumentCaptor.forClass(Map.class);
+        verify(notificationClient).sendEmail(
+                eq(PAYMENT_STOPPING.getTemplateId()), eq(currentCycle.getClaim().getClaimant().getEmailAddress()), argumentCaptor.capture(), any(), any());
+        assertChildTurnsFourEmailPersonalisationMap(currentCycle, argumentCaptor.getValue());
```

Comment:
The personalisation map will be different for the payment stopping email, we should change this.